libcloud/compute/drivers/azure_arm.py

```diff
@@ -706,6 +709,10 @@ def destroy_node(self, node, ex_destroy_nic=True, ex_destroy_vhd=True):
         this node (default True).
         :type node: ``bool``
 
+        :param ex_retries: Number of times to retry checking if the node is gone,
+        destroying the NIC or destroying the VHD.
```

Comment:
Please mention the default value.

Reply:
Now that I think more about it, I think `ex_retries` is misleading because it suggests that we retry destroying, but we only check if it's gone multiple times. Do you have an idea for a better name than `ex_retries` that would convey this?